src/cf-atomic-component/src/utilities/transition/ExpandableTransition.js

```diff
@@ -57,6 +55,13 @@ function ExpandableTransition( element, classes ) {
    */
   function _transitionComplete() {
     this.trigger( BaseTransition.END_EVENT, { target: this } );
```

Comment:
I think the lifecycle should be `transitionBegin` > `expandBegin` > `expandEnd` > `transitionEnd` …and… `transitionBegin` > `collapseBegin` > `collapseEnd` > `transitionEnd`. So this should go at the bottom of `_transitionComplete`. 

Making the above moot, I'm thinking the `transitionBegin` and `transitionEnd` events don't need to be dispatched from ExpandableTransition at all, since expand and collapse are the expandable-equivalent events. From the outside, when dealing with an ExpandableTransition it should not matter that it's using a BaseTransition under the hood. By exposing the BaseTransition events, it means that if they were ever changed in BaseTransition, it is no longer guaranteed that something depending on them through ExpandableTransition wouldn't break.

Reply:
I agree with the second point, updated.